drivers/pes/pswater.f90

```diff
@@ -612,9 +613,10 @@ subroutine dms_nasa(r1, q3, gradq)
 ath0 = 1.82400520401572996557d0
 costhe = -.24780227221366464506d0
 
-ROH1 = R1(:,2) - R1(:,1)
-ROH2 = R1(:,3) - R1(:,1)
-RHH  = R1(:,2) - R1(:,3)
+! MR: changed order of indexes to conform with what is now in pot_nasa and the array conventions of i-PI
+ROH1 = R1(2,:) - R1(1,:)
+ROH2 = R1(3,:) - R1(1,:)
```

Comment:
Yes, you are right, I thought those lines were from the "potential" subroutine and that you revert the edit I did... but, actually, you did the same kind of edit to a different subroutine...